### PR TITLE
New version: WasmEdge.WasmEdge version 0.13.2

### PR DESCRIPTION
**Pull request by abhinandanudupa**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112747&drop=dogfoodAlpha

### CODE REVIEW

**Comment by abhinandanudupa** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @abhinandanudupa,

This package requires a dependency:
image

**Comment by abhinandanudupa** [comment]
> Hi @abhinandanudupa,
> 
> This package requires a dependency: image

I am not sure about that, ig I had all the dependencies installed during development and thats why I did not notice them. But I don't find any issue in the issues section of the [project](https://github.com/WasmEdge/WasmEdge) related to this.

File: manifests/w/WasmEdge/WasmEdge/0.13.2/WasmEdge.WasmEdge.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: WasmEdge.WasmEdge
+PackageVersion: 0.13.2
+MinimumOSVersion: 10.0.0.0
+Installers:
```
Comment (stephengillie) [inline manifests/w/WasmEdge/WasmEdge/0.13.2/WasmEdge.WasmEdge.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (abhinandanudupa) [reply manifests/w/WasmEdge/WasmEdge/0.13.2/WasmEdge.WasmEdge.installer.yaml:11]: Thank you!

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

**Comment by abhinandanudupa** [comment]
@stephengillie Can this be merged now since version 1.6.2 has been released? Are any changed needed?

**Comment by abhinandanudupa** [comment]
@stephengillie Also please do take a look at these:

- https://github.com/microsoft/winget-pkgs/pull/112748
- https://github.com/microsoft/winget-pkgs/pull/112750

**Review by stephengillie — APPROVED** [review]

**Pull request merged**